torch/csrc/jit/codegen/cuda/lower_scalar_hoist.h

```diff
@@ -0,0 +1,100 @@
+#pragma once
```

Comment:
Looks like GitHub is not detecting this as a file renaming. There is nothing new here, except for some mechanical name changes like `index`->`value`